pkg/skaffold/render/renderer/render_mux_test.go

```diff
@@ -38,14 +38,14 @@ func TestRenderMux_Render(t *testing.T) {
 		expectedDeps []string
 		shouldErr    bool
 	}{
-		{
-			name: "concatenates render results with separator",
-			renderers: []Renderer{
-				mock{manifests: "manifest-1", deps: []string{"file1.txt", "file2.txt"}},
-				mock{manifests: "manifest-2", deps: []string{"file2.txt", "file3.txt"}}},
-			expected:     "manifest-1\n---\nmanifest-2",
-			expectedDeps: []string{"file1.txt", "file2.txt", "file3.txt"},
-		},
+		// {
```

Comment:
these tests appear to be passing at this PR currently, likely makes sense to uncommen them